demisto_sdk/commands/content_graph/parsers/generic_field.py

```diff
@@ -18,7 +18,7 @@ def __init__(
         super().__init__(path, pack_marketplaces, git_sha=git_sha)
         self.definition_id = self.json_data.get("definitionId")
         self.field_type = self.json_data.get("type") or ""
-
+        self.group = self.json_data.get("group")
```

Comment:
Let's add it as a property.
see cli_name in parsers/incident_field.py as a reference (make sure to add it to the field_mapping dict as well.